src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataApi.java

```diff
@@ -201,6 +201,20 @@ CaseDetails submitEventForCitizen(
             @RequestParam Map<String, String> searchCriteria
     );
 
+    /**
+     * Retrieve case by Case Id
+     */
+    @RequestMapping(
+            method = RequestMethod.GET,
+            value = "/cases/{cid}",
+            headers = CONTENT_TYPE + "=" + APPLICATION_JSON_VALUE
+    )
+    CaseDetails getCase(
+            @RequestHeader(AUTHORIZATION) String authorisation,
```

Comment:
have you tested that these headers are required? Wasn't able to see it in the API but they should be required

Reply:
Umm just included them since headers are ignored if not needed.

Reply:
we should test it first without to see what happens, have you tested the api with curl / this client?

Reply:
ill try without ...

Reply:
they are required ... probably got a request interceptor configured in spring 
I dont like the way you can hide that stuff away in spring ☹️

Reply:
you can do that in all frameworks, but normally you would inject it into the method so that you could use it... but they are aren't using it

Reply:
well the more functional ones are declarative in the routing so there are less surprises.